.travis.yml

```diff
@@ -27,4 +27,4 @@ after_success:
 - bash <(curl -s https://codecov.io/bash)
 
 go:
-- 1.8.3
+- 1.8.x
```

Comment:
One question is why we would need to use latest version of 1.8 than 1.8.3 as default.

Reply:
This will pick the latest. So it could be 1.8.4 so it's an okay patch. Not needed strictly.